src/software/sensor_fusion/vision_detection.cpp

```diff
@@ -0,0 +1,29 @@
+#include "software/sensor_fusion/vision_detection.h"
+
+VisionDetection::VisionDetection(
+    const std::vector<BallDetection> &ball_detections,
+    const std::vector<RobotDetection> &friendly_team_detections,
+    const std::vector<RobotDetection> &enemy_team_detections,
+    const Timestamp &latest_timestamp)
```

Comment:
Because each `Detection` class has its own timestamp, do we really need another timestamp here? It's not tied to any data in particular.

Reply:
fixed